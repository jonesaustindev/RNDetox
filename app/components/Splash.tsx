import React from 'react';
import {StatusBar, Text} from 'react-native';
import {SafeAreaView} from 'react-native-safe-area-context';

export function Splash() {
  return (
    <SafeAreaView
      style={{
        flex: 1,
        justifyContent: 'center',
        alignItems: 'center',
        backgroundColor: '#A267AC',
      }}>
      <StatusBar barStyle={'light-content'} />
      <Text style={{fontSize: 36, fontWeight: '600', color: '#ffffff'}}>
        RNTest
      </Text>
    </SafeAreaView>
  );
}
