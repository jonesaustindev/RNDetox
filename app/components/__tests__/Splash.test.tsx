import React from 'react';
import {render} from '@testing-library/react-native';
import {Splash} from '../Splash';

it('renders splash', () => {
  const loader = render(<Splash />);
  expect(loader).toMatchSnapshot();
});
