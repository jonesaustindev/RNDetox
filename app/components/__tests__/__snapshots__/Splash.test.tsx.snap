// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders splash 1`] = `
<RNCSafeAreaView
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#A267AC",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 36,
        "fontWeight": "600",
      }
    }
  >
    RNTest
  </Text>
</RNCSafeAreaView>
`;
